### ISSUE TITLE
Autofix for "inconsistently imported as implementation-only"

### ISSUE THREAD

**Issue by bimawa** [realm/SwiftLint#4634]
### New Issue Checklist

- [X] Updated SwiftLint to the latest version
- [X] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

We're creating a framework with closed binaries. And mark it like `@_implementationOnly import xxx`. But our teammates often forgot set this attribute. Our import go outside from library. We detect this issue only after release and when our client implemented our sdk.  Will be cool if swfitlint will automatically added this fix to imports which imported by implementationsOnly attributes.
`Rule Request: [impolementationOnly]`%

1. For prevent errors before the sell the sdk code;

I'm not sure is possible by swiftlint or not.

**Comment by SimplyDanny** [comment]
Could you please provide examples for cases that trigger the rule and case which don't trigger it?

**Comment by Cyberbeni** [comment]
You could create a custom rule: https://github.com/realm/SwiftLint#defining-custom-rules

If you need help with regex: https://regex101.com/

There is currently no autofix for these AFAIK.
